sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/schema.yaml

```diff
@@ -2346,3 +2346,6 @@ fields:
   - name: value
     type: INTEGER
     mode: NULLABLE
+- name: days_active_bits
```

Comment:
Should we add a description to this field to indicate what is meant by `active`?

Reply:
I added a description for both days_seen_bits  and days_active_bits.